src/components/molecules/PageHeader.tsx

```diff
@@ -21,7 +22,11 @@ export default function PageHeader({
   return (
     <header className={styleClasses}>
       <h1 className={styles.title}>{title}</h1>
-      {description && <p className={styles.description}>{description}</p>}
+      {description && (
+        <p className={styles.description}>
+          <Markdown text={description} />
```

Comment:
This results in invalid markup (can't have `p` add child of `p`):

<img width="455" alt="Screen Shot 2021-09-13 at 15 25 10" src="https://user-images.githubusercontent.com/90316/133091617-f7efb6f7-59eb-428e-b097-9e7cf0c79924.png">

So goal here would be to reduce nested markup as much as possible, ideally the `Markdown` component outputs as `p` and gets the `className={styles.description}`

Reply:
oh right, haven't seen that